src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -44,11 +45,12 @@
   private final ExportForms forms;
   private final ExportPanelForm form;
 
-  public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, ExportPanelForm form, BriefcasePreferences preferences) {
+  public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, ExportPanelForm form, BriefcasePreferences preferences, Analytics analytics) {
```

Comment:
This line is 155 characters. Do you have a limit? 😀

Reply:
The limit is your imagination! :D

It is good that this long line hurts. It's yelling at us that we need to rethink our dependencies. The bad fix IMO would be to place some line breaks...

Reply:
I’m amused—and maybe impressed—that you think it’s worse to break this line.